src/Sulu/Bundle/SecurityBundle/Resources/config/services.xml

```diff
@@ -39,6 +39,7 @@
     </parameters>
 
     <services>
+        <service id="sulu_security.encoder_factory" alias="security.password_hasher_factory" />
```

Comment:
We could remove the `sulu_security.encoder_factory` and replace it with `security.password_hasher_factory`

Reply:
And type it now as PasswordHasherFactoryInterface: https://github.com/sulu/sulu/blob/71d5cd1fd1ed3b4c1029e6e29a77ae07f7b758d4/src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php#L72 instead of the phpdoc